astronomer/providers/snowflake/example_dags/example_snowflake.py

```diff
@@ -66,8 +74,13 @@
         task_id="snowflake_op_sql_select_stmts", sql=SNOWFLAKE_SLACK_SQL, return_last=False
     )
 
+    snowflake_op_multiple_query_in_one_run = SnowflakeOperatorAsync(
```

Comment:
Is the objective of this task something different than the above `snowflake_op_sql_multiple_stmts` task?

Reply:
yes we basically want create, insert and drop in the same batch